metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/interfaces/MessageParser.java

```diff
@@ -34,15 +34,15 @@
    * @param rawMessage
    * @return If null is returned, this is treated as an empty list.
    */
-  List<T> parse(byte[] rawMessage);
+  List<T> parse(byte[] rawMessage, SensorParserConfig sensorParserConfig);
```

Comment:
> Regarding the similarity in names, I obviously don't think they're too similar (otherwise I'd not have suggested it ;).

Sorry, I don't know what is obvious to you and what is not.  It was not obvious to me that you had thought about that.